internal/engine/build_and_deployer_test.go

```diff
@@ -836,6 +837,18 @@ func (f *bdFixture) assertContainerRestarts(count int) {
 		"checking for expected # of container restarts")
 }
 
+// Total number of restarts, regardless of which container.
+func (f *bdFixture) assertTotalContainerRestarts(count int) {
+	// Ensure that MagicTestContainerID was the only container id that saw
+	// restarts, and that it saw the right number of restarts.
+	expected := map[string]int{}
```

Comment:
this appears to be a dead assignment? am i missing something?

Reply:
It gets called when you want to assert total number of restarts rather than asserting X restarts of `MagicTestContainerID`. I'm not in love with this naming/distinction but i didn't want to rewrite all the tests that use the latter logic.